### ISSUE TITLE
Number of comments in board view are not working.

### ISSUE THREAD

**Issue by tisto** [collective/collective.ploneboard#8]
Steps to reproduce:

1) Create a new Plone site
2) Create a new board
3) Add conversations and replies
4) Go back to the board view
-> The total comments column (currently commented out) always show 0 comments.

FYI: I commented out the  "# of conversations" column for now. See: https://github.com/collective/collective.ploneboard/commit/257ca0037a6ece4057472b281c86aa2e5587dd8d

**Comment by pigeonflight** [comment]
It seems that the counter inherited from plone.app.discussion only counts published comments.
this is documented in their doctest : https://github.com/plone/plone.app.discussion/blob/0a16de852664f0e33ceded2673931a9cec7f635b/plone/app/discussion/tests/functional_test_comment_review_workflow.txt#L191-L195

After publishing my board and topics (and uncommenting the line mentioned) I was able to get a comment count:
![image](https://user-images.githubusercontent.com/31827/49418288-6001ae00-f74f-11e8-9bd8-6a9dce4d0512.png)

**Comment by tisto** [comment]
@pigeonflight good catch! Yeah. I guess that makes sense. It would be cool if we could write an acceptance test for this (robot framework) and then re-add that functionality.
